### PR TITLE
Add kuvio as dependency

### PR DESCRIPTION
**Pull request by skeate**
Same caveat as #268

### CODE REVIEW

File: tests/EmailAddress.test.ts [hunk 1]
```diff
@@ -10,9 +10,6 @@ const valid: ReadonlyArray<string> = [
   'x@x.au',
   'foo@bar.com.au',
   'foo+bar@bar.com',
-  'hans.m端ller@test.com',
```
Comment (jacob-alford) [inline tests/EmailAddress.test.ts:13]: Are these actually invalid?  I think these came from validators js originally

File: tests/EmailAddress.test.ts [hunk 2]
```diff
@@ -31,10 +28,13 @@ const invalid: ReadonlyArray<string> = [
...
   '@invalid.com',
   'foo@bar.com.',
+  'hans.m端ller@test.com',
+  'hans@m端ller.com',
+  'test|123@m端ller.com',
   'somename@ｇｍａｉｌ.com',
   'foo@bar.co.uk.',
   'z@co.c',
-  // 'ｇｍａｉｌｇｍａｉｌｇｍａｉｌｇｍａｉｌｇｍａｉｌ@gmail.com', // passes (but shouldn't)
+  'ｇｍａｉｌｇｍａｉｌｇｍａｉｌｇｍａｉｌｇｍａｉｌ@gmail.com',
```
Comment (jacob-alford) [inline tests/EmailAddress.test.ts:37]: Great 👍🏻

File: tsconfig.json [hunk 3]
```diff
@@ -3,7 +3,7 @@
     "strict": true,
     "allowJs": false,
     "esModuleInterop": true,
-    "moduleResolution": "node",
+    "moduleResolution": "nodenext",
```
Comment (jacob-alford) [inline tsconfig.json:6]: 👍

**Review by jacob-alford — APPROVED** [review]
Thanks for integrating Kuvio!

File: tests/EmailAddress.test.ts (hunk 1 above)
Comment (skeate) [reply tests/EmailAddress.test.ts:13]: unquoted, yeah; from what I can tell reading the rfc, unquoted is only "printable US-ASCII characters not including specials"

https://datatracker.ietf.org/doc/html/rfc5322#section-3.2.3

**Pull request merged**